### PR TITLE
ann updates

### PR DESCRIPTION
**Pull request by Lewy09-Tm25**
ann model exploration updated. ann_results.csv is the result file. ann_models.csv was used to discover the fact that 250 epochs had the best results.

### CODE REVIEW

File: metrics/ann_results.csv [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Neurons,Accuracy,F1 Score,Recall,Precision,MCC,Kappa
+no hidden layer,0.787654321,0.762430939,0.821428571,0.711340206,0.577095562,0.57224965
+32,0.809876543,0.757097792,0.714285714,0.805369128,0.604756853,0.601830927
+64,0.8,0.773109244,0.821428571,0.73015873,0.59870749,0.595404595
+128,0.797530864,0.772222222,0.827380952,0.723958333,0.59572684,0.591452613
```
Comment (DanielYakubov) [inline metrics/ann_results.csv:5]: this has the best recall, is this what we will be using?

**Review by DanielYakubov — APPROVED** [review]
ok

File: notebooks/analysis of ann_models file.ipynb
Comment (DanielYakubov) [inline notebooks/analysis of ann_models file.ipynb:1]: give this a better name. Add _ don't use spaces - 
analysis_of_ann_models_file.ipynb

Comment (Lewy09-Tm25) [reply notebooks/analysis of ann_models file.ipynb:1]: Resolved.

File: metrics/ann_results.csv (hunk 1 above)
Comment (Lewy09-Tm25) [reply metrics/ann_results.csv:5]: yes.

**Pull request merged**